core/org.testeditor.core/src/main/java/org/testeditor/core/services/impl/TestProjectServiceImpl.java

```diff
@@ -199,10 +222,19 @@ public void unBind(TestServerService testServerService) {
 	private void loadProjectListFromFileSystem() throws SystemException {
 		testProjects = new ArrayList<TestProject>();
 		File[] files = getWorkspaceDirectories();
-		LOGGER.info("Scanning for Projects in: " + Platform.getLocation().toOSString());
+		logger.info("Scanning for Projects in: " + Platform.getLocation().toOSString());
 		for (File file : files) {
 			if (isTestProjectDirectory(file)) {
 				TestProject testProject = createProjectFrom(file);
+				try {
+					if (teamShareService.hasGlobalLock(testProject)) {
```

Comment:
Other services like the TestStructureService checks that an object is team shared before it calls the team service. this is based on the older implementation without the dispatcher. In this implementation the dispatcher is responsible to handle the no service is defined state. I think we should make this more clear if we decide this responsibility to the dispatcher. Maybe in a new user story.
What do you think?